modular_sojourn/spaceleak.dm

```diff
@@ -1,6 +1,6 @@
 
 /obj/item/bluespace_leak
-	name = "Leaking bluespace"
+	name = "leaking bluespace"
```

Comment:
Same with capitalization here. The wording doesn't roll off good either, it is also not what we usually would say like:
- Spacetime Rift
- Rift
- Portal
- Bluespace Leak
- Blue(space) Rift

Reply:
Will do this, was thinking back to the comment on decapitalized Bluespace Artillery on my closed PR, personally I think two capitalized is preferable.